ui/src/app/applications/components/utils.tsx

```diff
@@ -1031,7 +1035,9 @@ export function getPodStateReason(pod: appModels.State): {message: string; reaso
     }
 
     let initializing = false;
-    for (const container of (pod.status.initContainerStatuses || []).slice().reverse()) {
+    let i = -1;
+    for (const container of (pod.status.initContainerStatuses || []).slice()) {
+        i++;
```

Comment:
If we don’t reverse this array, we might not need the slice either. The slice is likely there to prevent modifying the order of the input data.

Also, if we need the loop index, using a for statement might be clearer than for...of:
```suggestion
    const initContainerStatuses = pod.status.initContainerStatuses || [];
    for (let i = 0; i < initContainerStatuses.length; i++) {
        const container = initContainerStatuses[i];
```

Reply:
Good idea. Done!